libs/ui/accordion/accordion.stories.ts

```diff
@@ -173,3 +174,29 @@ export const SetOpenState: Story = {
     `,
 	}),
 };
+export const WithTapable: Story = {
+	render: () => ({
+		template: `
+    <hlm-accordion>
+      <hlm-accordion-item>
+        <button hlmAccordionTrigger>
+          Is the button tapable when closed?
+          <hlm-icon hlmAccIcon />
+        </button>
+        <hlm-accordion-content>
+          <button data-testid='not-tapable-when-closed'>It should not be when closed</button>
+        </hlm-accordion-content>
+      </hlm-accordion-item>
+
+      <hlm-accordion-item>
+        <button hlmAccordionTrigger>
+          Is the button tapable when open?
+          <hlm-icon hlmAccIcon />
+        </button>
+        <hlm-accordion-content>
+          <button data-testid='tapable-when-open'>It should be when open</button>
+        </hlm-accordion-content>
+      </hlm-accordion-item>
+    `,
```

Comment:
Missing closing tag: `</hlm-accordion>`

Also, looking like Prettier doesn't run on storybook files?

<details>
<summary>Missing spaces with tabs (old & new)</summary>

Arrows are tabs and dots are spaces:

![image](https://github.com/goetzrobin/spartan/assets/19947758/a35ec8b0-7cbb-4918-af5c-5a050c20d7ef)


</details>